packages/suite/src/components/wallet/AccountsMenu/components/AccountItem/TokensCount.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useActions } from '@suite-hooks';
+import * as routerActions from '@suite-actions/routerActions';
+import { Button, variables, useTheme } from '@trezor/components';
+import { Translation } from '@suite-components/Translation';
+
+const ButtonBadge = styled(Button)`
+    margin: 0px 8px;
+    padding: 3px 4px;
+    font-size: 10px;
+    line-height: 1;
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    background: ${props => props.theme.STROKE_GREY_ALT};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+`;
+
+interface Props {
+    count: number;
+}
+
+export const TokensCount: React.FC<Props> = ({ count }) => {
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+    const theme = useTheme();
+    return (
+        <ButtonBadge
+            icon="PLUS"
+            variant="tertiary"
+            size={10}
+            color={theme.TYPE_LIGHT_GREY}
+            onClick={() => {
+                goto('wallet-tokens', undefined, true);
```

Comment:
I personally think the token badge should not be this clickable (you might discuss this with @matejzak), but I would not expect it there.

I recommend:
- disable hover color change for the token badge
- remove shortcut to tokens

If you and @matejzak decide to keep it clickable, please take a look at the buggy behaviour which is in place with current solution.  Steps to reproduce below.

Buggy behaviour - steps to reproduce:
- select account without tokens (e.g. BTC #1)
- then click on token badge in ETH account
- you will end up on "No tokens... yet." screen which is invalid for BTC and other accounts w/o tokens support

Reply:
Discussed with Ondra and @matejzak, they like the badge to be clickable, but I'm not sure about the hover color. And definitely I will fix the bug, thanks for the catch!